resources/entity/player/player.gd

```diff
@@ -27,6 +27,12 @@ func _ready() -> void:
 func _physics_process(delta: float) -> void:
 	super._physics_process(delta)
 	inputComponent.update(self)
+	
+	if direction.length() > 0:
+		$SubViewportContainer/SubViewport/AnimatedSprite2D.play("walk")
+	else:
+		$SubViewportContainer/SubViewport/AnimatedSprite2D.stop()
+	
```

Comment:
Is it necessary to call the play function each frame? If not, this logic should go into the input component i think.

Reply:
I don't know if it's smart to put it in the input component for two reasons:  
1. What if the player is moved in another way? Like pushed for example - do we want to play the animation there? If not -> Let's use the InputComponent, okay
2. Okay so let's say we put it in the input component unchanged. It would result in the same thing. The InputComponent get's updated every frame - the call is right above this if/else statement. So it really wouldn't make a difference and I personally believe that the player should turn on his animation

There is something that should definitely be done tho:  
Add a check if the animation is already running

Reply:
okay yeah I guess a check if the animation is playing in addition to the direction check would do fine.